frontend/packages/core/src/AppLayout/colors.tsx

```diff
@@ -0,0 +1,256 @@
+    400: "#F26E50",
+    500: "#F15534",
+    600: "#CA4428",
+    700: "#A1301C",
+    800: "#792111",
+    900: "#571608",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+};
+
+// TODO: Once colors are added to palettes, use them in choices
+export const LIGHT_MODE_CHOICES: ClutchColorChoices = {
+  typography: {
+    primary: LIGHT_MODE.neutral[900],
+    secondary: `${LIGHT_MODE.neutral[900]}A6`,
+    tertiary: `${LIGHT_MODE.neutral[900]}7A`,
+    inverse: "#FFFFFF",
+  },
+  stroke: {
+    primary: LIGHT_MODE.neutral[900],
+    secondary: `${LIGHT_MODE.neutral[900]}73`,
+    tertiary: `${LIGHT_MODE.neutral[900]}40`,
+    inverse: "#FFFFFF",
+  },
+  background: {
+    primary: "#FFFFFF",
+    secondary: LIGHT_MODE.blue[50],
+    header: "", // gradient
+  },
+  states: {
+    hover: {
+      active: `${LIGHT_MODE.blue[600]}0D`,
+      default: `${LIGHT_MODE.neutral[900]}0D`,
+    },
+    focused: {
+      active: `${LIGHT_MODE.blue[600]}1A`,
+      default: `${LIGHT_MODE.neutral[900]}1A`,
+    },
+    pressed: {
+      active: `${LIGHT_MODE.blue[600]}26`,
+      default: `${LIGHT_MODE.neutral[900]}21`,
+    },
+    selected: {
+      active: `${LIGHT_MODE.blue[600]}1F`,
+      default: `${LIGHT_MODE.neutral[900]}26`,
+    },
+    disabled: {
+      active: "", // blank on design system
+      default: `${LIGHT_MODE.neutral[900]}08`,
+    },
+  },
+};
+
+export const DARK_MODE: ClutchPalette = {
+  neutral: {
+    50: "#232542",
+    100: "#30324E",
+    200: "#3E4059",
+    300: "#55566D",
+    400: "#77788A",
+    500: "#8D8E9E",
+    600: "#A4A5B1",
+    700: "#D2D2D8",
+    800: "#E8E8EB",
+    900: "#F8F8F9",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  blue: {
+    50: "#090A42",
+    100: "#050656",
+    200: "#0A086B",
+    300: "#13199D",
+    400: "#2A4FF6",
+    500: "#4281F6",
+    600: "#5AABF6",
+    700: "#8CC4F8",
+    800: "#C2E1FE",
+    900: "#DCECFB",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  green: {
+    50: "#002C05",
+    100: "#084713",
+    200: "#145F1D",
+    300: "#217A2A",
+    400: "#2D9638",
+    500: "#54B45B",
+    600: "#73C178",
+    700: "#9CD29E",
+    800: "#C3E4C4",
+    900: "#E6F4E7",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  amber: {
+    50: "#352215",
+    100: "#7E3E02",
+    200: "#985304",
+    300: "#B26A08",
+    400: "#CC820D",
+    500: "#E69F2A",
+    600: "#EAB04E",
+    700: "#EFC67F",
+    800: "#F6DCB1",
+    900: "#FBF1E0",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+  red: {
+    50: "#501306",
+    100: "#621809",
+    200: "#751E0D",
+    300: "#972814",
+    400: "#B3351E",
+    500: "#C8482C",
+    600: "#EB7A60",
+    700: "#F4A08A",
+    800: "#F0B9AB",
+    900: "#FBE8E7",
+    A100: "",
+    A200: "",
+    A400: "",
+    A700: "",
+  },
+};
+
+export const DARK_MODE_CHOICES: ClutchColorChoices = {
+  typography: {
+    primary: `${DARK_MODE.neutral[900]}E6`,
```

Comment:
@dschaller just saw the comment about using alpha() in mui5. Should I keep the alpha as a separate field here? Like:
primary: color
alpha: .9

etc. ?

Reply:
did some research, didn't really come to a good conclusion though. I wonder if we would be better off organizing our colors like this? https://mui.com/customization/default-theme/?expand-path=$.palette (also they use rgba(x,y,z,o) instead of just putting everything in hex). Or having the palette follow the format like here https://mui.com/customization/palette/